app/assets/javascripts/discourse/app/lib/text-direction.js

```diff
@@ -13,19 +13,22 @@ export function isLTR(text) {
   return ltrDirCheck.test(text);
 }
 
-export function setTextDirections($elem) {
-  $elem.find("*").each((i, e) => {
-    let $e = $(e),
-      textContent = $e.text();
+export function setTextDirections(elem) {
+  for (let e of elem.children) {
+    const textContent = e.textContent;
     if (textContent) {
-      isRTL(textContent) ? $e.attr("dir", "rtl") : $e.attr("dir", "ltr");
+      isRTL(textContent)
```

Comment:
Maybe a small tweak since we're here? 😃 No variable, and two calls merged:

```js
     if (e.textContent) {
       e.setAttribute("dir", isRTL(e.textContent) ? "rtl" : "ltr");
     }
```